### PR TITLE
[Do not review] Enabling selection for input changed for UnifiedPicker and updating example for that case

### PR DESCRIPTION
**Pull request by nebhatna**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx [hunk 1]
```diff
@@ -270,12 +270,25 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       props.inputProps.onClick(ev as React.MouseEvent<HTMLInputElement>);
     }
   };
-  const _onInputChange = (value: string, composing?: boolean) => {
+  const _onInputChange = (value: string, composing?: boolean, resultItemsList?: T[]) => {
```
Comment (elisabethcvs) [inline packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx:273]: Would this ever have a value?

File: packages/react-examples/src/experiments/UnifiedPeoplePicker/DoubleUnifiedPeoplePicker.Example.tsx [hunk 2]
```diff
@@ -179,14 +161,54 @@ const UnifiedPeoplePickerExample = (): JSX.Element => {
...
     const suggestionList = suggestions.map(item => {
       return { item: item, isSelected: false, key: item.key } as IFloatingSuggestionItem<IPersonaProps>;
     });
-    // We want to show top 5 results
-    setPeopleSuggestions(suggestionList.splice(0, 5));
+
+    const updatedItems: IPersonaProps[] = [];
+    const currentItems: IPersonaProps[] = [...peopleSelectedItems];
+
+    for (let i = 0; i < currentItems.length; i++) {
```
Comment (elisabethcvs) [inline packages/react-examples/src/experiments/UnifiedPeoplePicker/DoubleUnifiedPeoplePicker.Example.tsx:184]: I don't think you need this, you can have the declaration be this right?
const updatedItems: IPersonaProps[] = [...peopleSelectedItems];

**Comment by elisabethcvs** [comment]
There's paste logic in the other two examples, should those get updated as well?

File: packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx (hunk 1 above)
Comment (nebhatna) [reply packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx:273]: yes, it would, this is what is getting updated

**Pull request closed without merging**